### DISCUSSION TITLE
Sample projects of Reactive Stream support

### DISCUSSION THREAD

**Discussion by taxone** [spring-projects/spring-integration#3443] [Q&A] [answered]
Hi,
I'm looking for some sample projects with examples about the Reactive Streams support provided  by Spring Integration.
I looked [at the offical site of Spring Integration samples](https://github.com/spring-projects/spring-integration-samples), but I did not find anything.
Thanks.

**Comment by artembilan** [comment, marked as answer]
Hi!

Unfortunately we don't have enough resources in project team to cover all the samples aspects.
Also it is not going to be so simple sample since reactive programming is not just the way to show bridging from `Flux` to Spring Integration flow and back. We would need to build the whole end-to-end complex solution to demonstrate all the power of reactive API in Spring Integration with its utilities and channel adapters.

The best place at the moment to obtain some reactive info in Spring Integration is documentation: https://docs.spring.io/spring-integration/docs/current/reference/html/reactive-streams.html#reactive-streams

We won't mind adding some samples if there is a good simple general idea to satisfy every single seeker.
